tests/test_environment.py

```diff
@@ -160,6 +165,8 @@ def test_era5_atmosphere(mock_show):
 def test_gefs_atmosphere(mock_show, example_env_robust):
     example_env_robust.setAtmosphericModel(type="Ensemble", file="GEFS")
     assert example_env_robust.allInfo() == None
+    assert isinstance(example_env.allPlotInfoReturned(), dict)
+    assert isinstance(example_env.allInfoReturned(), dict)
```

Comment:
The tests just checks if these methods return a dict? Wouldn't it be better to check if the dict has the keys and values that we expect from it?

Reply:
Good point, solved.